openshift/e2e-common.sh

```diff
@@ -208,6 +208,10 @@ function run_e2e_rekt_tests(){
   oc -n knative-eventing set env deployment/mt-broker-controller BROKER_INJECTION_DEFAULT=true || return 1
   wait_until_pods_running $EVENTING_NAMESPACE || return 2
 
+  echo "Replacing knative-eventing-test-event-library"
+  sed -i -e "s|registry.ci.openshift.org/openshift/knative-.*:knative-eventing-test-event-library|${KNATIVE_EVENTING_TEST_EVENT_LIBRARY}|g" "$(dirname "$0")/../test/rekt/resources/eventlibrary/eventlibrary.yaml"
+  echo "Replaced: $(dirname "$0")/../test/rekt/resources/eventlibrary/eventlibrary.yaml"
+  cat "$(dirname "$0")/../test/rekt/resources/eventlibrary/eventlibrary.yaml"
```

Comment:
I think we should do that for all `rekt test` images

Reply:
Yep, I was checking if this works or not. I will do it for all test images